src/client/components/ImageMenu.js

```diff
@@ -28,6 +29,8 @@ const ImageMenu = ({
   imageTitle,
   onSignatures,
 }) => {
+  const [activeImageMenu, toggleImageMenu] = useState('')
```

Comment:
I don't think we need to name the image menu using the `imageTag` prop. Since each instance of `ImageMenu` on the page will maintain its own internal state, we can use an easier to understand concept like `const [active, setActive] = setState(false)`

Reply:
Done!